src/quo2/components/avatars/account_avatar/style.cljs

```diff
@@ -0,0 +1,50 @@
+(ns quo2.components.avatars.account-avatar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def default-size 80)
+
+(defn get-border-radius
+  [size]
+  (case size
+    80 16
+    48 12
+    32 10
+    28 8
+    24 8
+    20 6
+    16 4))
+
+(defn get-emoji-size
+  [size]
+  (case size
+    80 36
+    48 24
+    32 15
+    28 12
+    24 11
+    20 11
+    16 11))
```

Comment:
This `size` param is passed directly from the component props, it means if we pass an invalid size while calling the component this `case` statement will crash with an exception. So please add the default value.

Also, I find a little hard passing a number for size, I mean, we have a set of valid sizes and any other number will fail, so maybe we could create a set of keywords (small, medium, or sm, md, etc), but I know this doesn't match with figma, so is hard.

Wdyt?

@J-Son89 @ilmotta

Reply:
I agree with the sentiment @ulisesmac. The problem is that the DS is built with non-semantic spaces and sizes. If we can convince them to change how they manage such a foundational aspect of their work than it would help create a more maintainable DS, but good luck convincing them about this 🤷🏼 It was always shocking to me that the entire app is designed without a grid system in mind. Many sizes/spaces are chosen arbitrarily (their own words), although they do try to follow some patterns, like multiples of 4.

As you hinted, we could come up with a standardized list of sizes and each component could use those instead of numbers, like what's used in Tailwind https://tailwindcss.com/docs/responsive-design. I actually like this and have commented a few times with @J-Son89. But this can also cause some confusing and mental effort to translate them to what's used in Figma and can also increase the likelihood of bugs. Since now we sort of want to just mimic what's in Figma to avoid these erroneous code<->Figma translations, the numbers might be okay for now.

On the other hand, it's a serious technical mistake to build the mobile app without responsiveness in mind from the get go. And responsiveness requires us to not hardcode magical numbers. It'll be very expensive (development-wise) to do this in the future. Flip phones and foldable phones are a thing now, and obviously tablets. Screen sizes are not that fixed little world of iPhone 11 used by the DS. It's just 😢. I know of many users who love their tablets and would like to use a "super app" such as Status in their bigger screens. Not to count many users like to read with their phones on the horizontal. Anyway, responsiveness and semantic sizes seem to be distant in the future...

> This size param is passed directly from the component props, it means if we pass an invalid size while calling the component this case statement will crash with an exception. So please add the default value.

I really miss clj-kondo not having a rule for forcing a default result in `case`. I'm considering creating a new linter for this. I really don't see any benefit in risking throwing exceptions, so to me the rule is simple, we should always add a default value in `case`.

Reply:
> Also, I find a little hard passing a number for size, I mean, we have a set of valid sizes and any other number will fail, so maybe we could create a set of keywords (small, medium, or sm, md, etc), but I know this doesn't match with figma, so is hard.

I understand @ulisesmac, It's a bit painful, I'm not a fan of numbers too as you mentioned it's error-prone, but we have to maintain the synchronisation between Figma and Quo2 implementation. I added a default value.

<hr />

> On the other hand, it's a serious technical mistake to build the mobile app without responsiveness in mind from the get go. And responsiveness requires us to not hardcode magical numbers. It'll be very expensive (development-wise) to do this in the future. Flip phones and foldable phones are a thing now, and obviously tablets. Screen sizes are not that fixed little world of iPhone 11 used by the DS. It's just 😢. I know of many users who love their tablets and would like to use a "super app" such as Status in their bigger screens. Not to count many users like to read with their phones on the horizontal. Anyway, responsiveness and semantic sizes seem to be distant in the future...

I don't encourage using pixel value as it's not responsive and will have different effects based on the DPI/PPI of the user's device. As you mentioned, it will become more tedious to refactor if we decide to be responsive in future. Can't argue 😢. This is more of a team discussion.

Reply:
> I don't encourage using pixel value as it's not responsive and will have different effects based on the DPI/PPI of the user's device. As you mentioned, it will become more tedious to refactor if we decide to be responsive in future. Can't argue cry. This is more of a team discussion.

For sure @smohamedjavid, this is a team-level discussion. I brought up to see how you guys reacted to it and if we're more on less on the same page. Nothing that blocks your PR.